lib/main.js

```diff
@@ -44,8 +44,8 @@ function accumulator({
   return calculateAccumulator(selections, stake, ew, idx, place);
 }
 
-function calculateMultiple(selections, stake, ew = false, fullCover = false) {
-  const count = selections.length;
+function calculateMultiple(selections, stake, ew = false, fullCover = false, size = null) {
+  const count = size || selections.length;
```

Comment:
Can you define `selections.length` as the default param for size?

Edit: In fact, if you define `count = selections.length` in the args, you don't need the `count` declaration inside the function at all

Reply:
👍